src/Features/CSharp/Portable/InlineHints/CSharpInlineTypeHintsService.cs

```diff
@@ -80,6 +80,15 @@ public CSharpInlineTypeHintsService()
                 }
             }
 
+            if (node is ImplicitObjectCreationExpressionSyntax { Parent: ArgumentSyntax } implicitNew)
```

Comment:
Should we check for `InlineHintsOptions.ForObjectCreationParameters || displayAllOverride` or should this get its own option?

Reply:
1. this should get it's own option
2. this should just check the option, and not check if parented by an argument.  i.e. `var v = x ? new List<int>() : new() { }` should show it there.

Reply:
> 1.

Added in c8dda544

> 2.

Done in d39651e8